release-process.md

```diff
@@ -186,6 +186,18 @@ that looks something like `[VOTE][RESULT] ...`.
 
 <h3>Finalize the release</h3>
 
+Note that `dev/create-release/do-release-docker.sh` script (`finalize` step ) automates most of the following steps **except** for:
+- Publish to CRAN
+- Update the configuration of Algolia Crawler
+- Remove old releases from Mirror Network
+- Update the rest of the Spark website
+- Create and upload Spark Docker Images
+- Create an announcement
+
+Please manually verify the result after each step.
+
+<h4>Upload to Apache release directory</h4>
```

Comment:
Isn't this step also manual?

Reply:
"Moving Spark binaries to the release directory" is automated.

Reply:
See more at https://github.com/xinrong-meng/spark/blob/master/dev/create-release/release-build.sh#L97. : )

Reply:
Oh, it was changed by SPARK-35872. 👍